Sorting/mergeSort.js

```diff
@@ -0,0 +1,48 @@
+// i am really not sure if this mergeSort works correctly. i think it does, because there is only one loop statement in the entire function-pair
+// which suggests that runtime would be less than < O(n^2).
+// but i am not sure if my basecases (arr.length === 1) and (arr.length === 2) are both necessary, or if only one is required.
+// also, with respect to memory, i am not sure if creating a mergedSorted[] in the helper function counteracts any efficiency gained with the recursive definition
+
+function mergeSort(arr) {
+  if (arr.length <= 0) {
+    console.log('Please be reasonable.');
```

Comment:
haha! 

(note: this condition can be combined with the one below. an empty array is sorted, just as an array of length 1 is sorted)